src/libstd/sys/windows/c.rs

```diff
@@ -638,6 +635,34 @@ if #[cfg(not(target_vendor = "uwp"))] {
     pub type PVECTORED_EXCEPTION_HANDLER = extern "system"
             fn(ExceptionInfo: *mut EXCEPTION_POINTERS) -> LONG;
 
+    pub const HANDLE_FLAG_INHERIT: DWORD = 0x00000001;
```

Comment:
These 4 items simply moved up a little bit in the file because other items moved out. These are still forbidden when targeting UWP.